Crtx 236720 aws security hub v2 (#44871)
demisto/content: Packs/AWS-SecurityHub/ReleaseNotes/1_4_0.md

#### Integrations

##### New: AWS - Security Hub v2

> Added the AWS Security Hub V2 integration, which manages unified, OCSF-formatted security and compliance findings using the AWS Security Hub V2 API. This integration fetches AWS Security Hub V2 findings as Cortex XSOAR incidents, with configurable first fetch time, maximum incidents per fetch, minimum severity, and additional OCSF field filters.

Reviewer: ```suggestion
Added the AWS Security Hub V2 integration, enabling ingestion and management of unified security and compliance findings in OCSF format through the AWS Security Hub V2 API. The integration imports AWS Security Hub V2 findings into Cortex with configurable first fetch time, maximum findings per fetch, minimum severity, and OCSF field-based filtering. It also supports bidirectional mirroring of finding updates between AWS Security Hub V2 and Cortex XSOAR.
```

- Added the following commands:
  - ***aws-securityhub-v2-security-hub-enable***
  - ***aws-securityhub-v2-security-hub-disable***
  - ***aws-securityhub-v2-findings-get***
  - ***aws-securityhub-v2-findings-batch-update***
- Added support for incident mirroring between AWS Security Hub and Cortex XSOAR.
- Added incoming mirroring: Each mirrored finding is re-fetched by its OCSF finding UID and changes (including severity) are reflected in the Cortex XSOAR incident. The mirrored severity reflects the finding's current severity as shown in the AWS Security Hub console.
- Added incoming lifecycle sync: When a finding is set to *Resolved* or *Suppressed* in AWS Security Hub, the corresponding Cortex XSOAR incident is closed. When it returns to *New* or *In Progress*, the incident is reopened.
- Added outgoing mirroring: Severity, status, and comment changes made in Cortex XSOAR are pushed back to the finding in AWS Security Hub. Changing the incident's severity (including the built-in ***Severity*** field) is mirrored to the finding's severity in AWS Security Hub.
- Added the ***Incident Mirroring Direction*** parameter.

> - Added the ***Resolve finding of closed incident from Cortex XSOAR in AWS Security Hub*** parameter.

Reviewer: No need to add all the explanation about the mirroring process, it is explained in the README.

#### Integrations

##### New: AWS - Security Hub v2

> Added a new integration for AWS Security Hub V2, which manages unified, OCSF-formatted security and compliance findings using the AWS Security Hub V2 API.

Reviewer 2: ```suggestion
Added the AWS Security Hub V2 integration, which manages unified, OCSF-formatted security and compliance findings using the AWS Security Hub V2 API. This integration fetches AWS Security Hub V2 findings as Cortex XSOAR incidents, with configurable first fetch time, maximum incidents per fetch, minimum severity, and additional OCSF field filters.
```

@ilaredo specific for Cortex XSOAR? Not relevant for XSIAM?

> - Added the ability to fetch AWS Security Hub V2 findings as Cortex XSOAR incidents, with configurable first fetch time, maximum incidents per fetch, minimum severity, and additional OCSF field filters.

Reviewer 2: ```suggestion
```

- Added the following commands:
  - ***aws-securityhub-v2-security-hub-enable***
  - ***aws-securityhub-v2-security-hub-disable***
  - ***aws-securityhub-v2-findings-get***
  - ***aws-securityhub-v2-findings-batch-update***

> - Added support for incident mirroring between AWS Security Hub and Cortex XSOAR.

Reviewer 2: @ilaredo Cortex or Cortex XSOAR?

Author: Cortex XSOAR

>   - Added incoming mirroring: each mirrored finding is re-fetched by its OCSF finding UID and changes (including severity) are reflected in the Cortex XSOAR incident. The mirrored severity reflects the finding's current severity as shown in the AWS Security Hub console.

Reviewer 2: ```suggestion
  - Added incoming mirroring: Each mirrored finding is re-fetched by its OCSF finding UID and changes (including severity) are reflected in the Cortex XSOAR incident. The mirrored severity reflects the finding's current severity as shown in the AWS Security Hub console.
```

>   - Added incoming lifecycle sync: when a finding is set to *Resolved* or *Suppressed* in AWS Security Hub, the corresponding Cortex XSOAR incident is closed. When it returns to *New* or *In Progress*, the incident is reopened.

Reviewer 2: ```suggestion
  - Added incoming lifecycle sync: When a finding is set to *Resolved* or *Suppressed* in AWS Security Hub, the corresponding Cortex XSOAR incident is closed. When it returns to *New* or *In Progress*, the incident is reopened.
```

>   - Added outgoing mirroring: severity, status, and comment changes made in Cortex XSOAR are pushed back to the finding in AWS Security Hub. Changing the incident's severity (including the built-in `Severity` field) is mirrored to the finding's severity in AWS Security Hub.

Reviewer 2: ```suggestion
  - Added outgoing mirroring: Severity, status, and comment changes made in Cortex XSOAR are pushed back to the finding in AWS Security Hub. Changing the incident's severity (including the built-in `Severity` field) is mirrored to the finding's severity in AWS Security Hub.
```

- Added the `Incident Mirroring Direction` parameter.
- Added the `Resolve finding of closed incident from Cortex XSOAR in AWS Security Hub` parameter.

#### Mappers

##### New: AWS Security Hub v2 - Incoming Mapper

> Added a new incoming mapper that maps OCSF-formatted findings to Cortex XSOAR incident fields for incoming mirroring, including finding details, severity, status, resources, remediation, and compliance information.

Reviewer 2: ```suggestion
Added an incoming mapper that maps OCSF-formatted findings to Cortex XSOAR incident fields for incoming mirroring, including finding details, severity, status, resources, remediation, and compliance information.
```

##### New: AWS Security Hub v2 - Outgoing Mapper

> Added a new outgoing mapper that maps Cortex XSOAR incident fields (severity, status, and comment) to AWS Security Hub V2 finding fields for outgoing mirroring.

Reviewer 2: ```suggestion
Added an outgoing mapper that maps Cortex XSOAR incident fields (severity, status, and comment) to AWS Security Hub V2 finding fields for outgoing mirroring.
```
